package.json

```diff
@@ -20,14 +20,14 @@
     "@ionic/core": "^8.0.0-beta.3"
   },
   "devDependencies": {
-    "@stencil/core": "^2.22.2",
-    "@types/jest": "^26.0.24",
+    "@stencil/core": "^4.14.0",
+    "@types/jest": "^29.5.12",
     "@typescript-eslint/eslint-plugin": "^5.0.0",
     "@typescript-eslint/parser": "^5.0.0",
     "eslint": "^8.0.0",
-    "jest": "^26.6.3",
-    "jest-cli": "^26.6.3",
-    "puppeteer": "^10.4.0",
+    "jest": "^29.7.0",
+    "jest-cli": "^29.7.0",
+    "puppeteer": "^21.11.0",
```

Comment:
~ Puppeteer should be an optional dependency now, if we do not have e2e tests in this project.